curriculum/challenges/english/25-front-end-development/workshop-reusable-mega-navbar/6745e2aca895c935168e7d91.md

```diff
@@ -0,0 +1,173 @@
+---
+id: 6745e2aca895c935168e7d91
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+To finish things up, set a text of `Calender` for the first anchor tag, `Chat` for the second, and `Email` for the third one.
+
+With that, you mega navbar component is complete!
+
+# --hints--
+
+You should set a text of `Calender` for the first anchor element.
+
+```js
+assert(document.querySelectorAll('li a')[3]?.textContent === 'Calender');
+```
+
+You should set a text of `Chat` for the second anchor element.
+
+```js
+assert(document.querySelectorAll('li a')[4]?.textContent === 'Chat');
+```
+
+You should set a text of `Email` for the third anchor element.
+
+```js
+assert(document.querySelectorAll('li a')[5]?.textContent === 'Email');
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Hello World</title>
+    <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
+    <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+
+    <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+    <script
+      data-plugins="transform-modules-umd"
+      type="text/babel"
+      src="index.jsx"
+    ></script>
+     <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div id="root"></div>
+    <script
+      data-plugins="transform-modules-umd"
+      type="text/babel"
+      data-presets="react"
+      data-type="module"
+    >
+      import { Navbar } from './index.jsx';
+      ReactDOM.createRoot(document.getElementById('root')).render(
+        <Navbar />
+      );
+    </script>
+  </body>
+</html>
+```
+
+```css
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --white: #fff;
+  --light-grey: #e1e0e0;
+  --dark-purple: #7c0e7c;
+  --black: #000;
+}
+
+body {
+  background-color: var(--light-grey);
+}
+
+.navbar {
+  background-color: var(--white);
+}
+
+.navbar ul {
+  display: flex;
+  justify-content: space-around;
+}
+
+.navbar ul li {
+  list-style: none;
+  border-radius: 4px;
+}
+
+.navbar ul li a {
+  text-decoration: none;
+  color: var(--black);
+  padding: 10px;
+  display: inline-block;
+  width: 100%;
+}
+
+.navbar ul .nav-item a:hover {
+  background-color: var(--dark-purple);
+  color: var(--white);
+}
+
+.navbar ul .nav-item .sub-menu {
+  visibility: hidden;
+  opacity: 0;
+  position: absolute;
+  right: 5%;
+  transition: opacity 0.5s ease;
+  display: block;
+  background-color: var(--white);
+}
+
+@media (min-width: 768px) {
+  .navbar ul .nav-item .sub-menu {
+    right: 15%;
+  }
+}
+
+@media (min-width: 1024px) {
+  .navbar ul .nav-item .sub-menu {
+    right: 13%;
+  }
+}
+
+.navbar ul .nav-item:hover .sub-menu,
+.navbar ul .nav-item:focus-within .sub-menu {
+  visibility: visible;
+  opacity: 1;
+}
+```
+
+```jsx
+export const Navbar = () => {
+  return (
+    <nav class="navbar">
```

Comment:
this should be `className` not `class`.
you will need to update this in all instances where this occurs